src/en/developer-layers-interfaces.md

```diff
@@ -39,230 +39,47 @@ this interface?
 - What states (if any) should this interface raise on the requirer?
 
 
-## What communication scopes are, and how to use them
+## The Relation Lifecycle
 
-When writing an interface, there is also the concept of a communication scope.
-There are three distinct flavors of scoping for a conversation. At times there
-will be fairly static information being transmitted between services - and this
-is a prime candidate for a [GLOBAL](#global) scope. If the information varies
-from service to service but remains the same for each unit of the service, you
-will want to investigate [SERVICE](#service) level conversations. The final, and
-default communication scope is [UNIT](#unit) level conversations, where each
-unit in a service group gets its own conversation with the provider.
+Before writing any interface code, its also important to understand the
+relationship lifecycle and how this impacts the communication events happening
+between units participating in the conversation.
 
-#### GLOBAL
+### Relation Scopes
 
-All connected services and units for this relation will share a single
-conversation. The same data will be broadcast to every remote unit, and
-retrieved data will be aggregated across all remote units and is expected to
-either eventually agree or be set by a single leader.
+An interfaces relationship scope falls into one of two categories. An implied
+`global` scope, which is the default for all interfaces. Or a `container` scope
```

Comment:
Capitalize the relationship scope names here to match what is in the next document (GLOBAL, CONTAINER).  These would also be great links to the next document with the appropriate anchors.